lib/smtp_server/sendgrid.dart

```diff
@@ -0,0 +1,5 @@
+import '../smtp_server.dart';
+
+SmtpServer sendgrid(String username, String password) =>
+    SmtpServer('smtp.sendgrid.net',
+        username: username, password: password, allowInsecure: true);
```

Comment:
why did you select `allowInsecure`?

Shouldn't sendgrid accept encrypted connections?
https://sendgrid.com/docs/for-developers/sending-email/getting-started-smtp/

Reply:
ping